src/App.vue

```diff
@@ -1,10 +1,24 @@
 <template>
   <v-app class="app-container" id="app">
-    <!-- FUTURE: dialogs go here -->
+    <!-- Dialogs -->
+    <NameRequestInvalidErrorDialog
+      :dialog="nameRequestInvalidErrorDialog"
+      @okay="nrOkay()"
+      @redirect="redirectToBusinessUrl()"
+      :type="nameRequestInvalidType"
+      attach="#app"
+    />
+
+    <AccountAuthorizationDialog
+      :dialog="accountAuthorizationErrorDialog"
+      @exit="redirectToBusinessUrl()"
+      @retry="evaluateNRPreconditions()"
+      attach="#app"
+    />
 
     <!-- Initial Page Load Transition -->
     <transition name="fade">
-      <div class="loading-container fade-out">
+      <div class="loading-container fade-out" v-show="evaluatingPreconditions">
```

Comment:
You need to also remove the "fade-out" class (which has a hard-coded transition delay).
See how it's implemented here: https://github.com/bcgov/business-filings-ui/blob/b86d130857692d92f402245abc517e6dc3ed9cd5/src/views/Correction.vue#L40

Reply:
Updated to remove fade-out